src/sbvr-api/permissions.ts

```diff
@@ -1477,11 +1462,6 @@ export const resolveAuthHeader = async (
 	// TODO: Consider making tx the second argument in the next major
```

Comment:
@thgreasi ?

Reply:
Same as below
> that's just imo that `req, tx, paramName` makes more sense, since both req & tx provide context, while `param` is the real arguments. I'm fine with it as is though.